kalitectl.py

```diff
@@ -80,7 +80,13 @@
 # Necessary for loading default settings from kalite
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "kalite.settings")
 
-KALITE_HOME = os.path.join(os.path.expanduser("~"), ".kalite")
+from django.conf import settings
```

Comment:
You can't do this here :) Well, maybe in practice it will work :) ...but we should separate loading the whole django stack from the main scope of kalitectl, otherwise we are taking a functional django env. for granted which will not be the case when using commands that configure stuff.